mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -480,10 +638,14 @@ object LinearRegression extends DefaultParamsReadable[LinearRegression] {
 class LinearRegressionModel private[ml] (
     @Since("1.4.0") override val uid: String,
     @Since("2.0.0") val coefficients: Vector,
-    @Since("1.3.0") val intercept: Double)
+    @Since("1.3.0") val intercept: Double,
+    @Since("2.3.0") val scale: Double)
```

Comment:
If scale is not used in the prediction process, I would suggest not to put it into the model.

Reply:
This is suggested by @jkbradley , and I also agree that it should be useful for model interpretation and debugging. And I have provided another constructor for linear regression model produced by _squared error_.